src/modules/components/vote/voteDataListItem/voteDataListItemStructure/voteDataListItemStructure.tsx

```diff
@@ -0,0 +1,76 @@
+import classNames from 'classnames';
+import {
+    DataList,
+    NumberFormat,
+    Tag,
+    formatterUtils,
+    type IDataListItemProps,
+    type TagVariant,
+} from '../../../../../core';
+import { type ICompositeAddress } from '../../../../types';
+import { addressUtils } from '../../../../utils';
+import { MemberAvatar } from '../../../member';
+
+export interface IVoteDataListItemStructureProps extends IDataListItemProps {
```

Comment:
The component is currently missing the handling of the `you` and `delegate` tags, a similar handling has already been implemented for the `MemberDataListItem` component